### ISSUE TITLE
cannot edit found and snatched movies

### ISSUE THREAD

**Issue by claudeguy** [nosmokingbandit/watcher#66]
on mac os x i can only edit movies with the state "wanted".  when i click on "found" or "snatched" movies i only see the dark background of the light window.

**Comment by nosmokingbandit** [comment]
Send me a copy of your watcher log.

**Comment by claudeguy** [comment]
here's the log file [log.txt](https://github.com/nosmokingbandit/watcher/files/679608/log.txt)

**Comment by scalda** [comment]
Same problem. here is my log.  [log](http://pastebin.com/X5nw9VDF)

**Comment by nosmokingbandit** [comment]
This is hard to figure exactly from the logs, but when building the html it fails when getting source indexer for the found nzbs.

The easiest way to see what the problem is would be to remove and re-add the movie. Which you can't do.

Could you send me a copy of your watcher.sqlite files? They don't have any critical info in them, I just need to look the list of search results and see what isn't cooperating.

**Comment by bbobrian** [comment]
Same here as well:
ValueError: None not a tag or string.

**Comment by claudeguy** [comment]
thanks for looking into it! hope you find what you need in the watcher.sqlite file: 
[watcher.sqlite.zip](https://github.com/nosmokingbandit/watcher/files/683005/watcher.sqlite.zip)

**Comment by dg0664** [comment]
Same issue here on OS X using Safari or any other browser.

**Comment by f3st3r113** [comment]
I am also experiencing this issue both on OS X and Ubuntu 16.04

**Comment by nosmokingbandit** [comment]
Do all of you use nzb.su?

**Comment by dg0664** [comment]
I do.

**Comment by nosmokingbandit** [comment]
That seems to be the problem...

nzb.su doesn't put their name in the `<title>` element of the XML.

I don't have an nzb.su account, so I'll need some help.

Someone enter this in their browser:

`https://www.nzb.su/api?apikey=XXX&t=movie&imdbid=0947798`

and replace XXX with your api key.

The result will look like this:

``
<?xml version="1.0" encoding="utf-8" ?> 
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:newznab="http://www.newznab.com/DTD/2010/feeds/attributes/">
<channel>
<atom:link href="https://6box.me/api?apikey=XXX&amp;t=movie&amp;imdbid=0947798" rel="self" type="application/rss+xml" />
<title>6box</title>
<description>6box Feed</description>
<link>https://6box.me/</link>
<language>en-gb</language>
<webMaster>6box.me@gmail.com (6box)</webMaster>
<category></category>
<image>
	<url>https://6box.me/templates/default/images/banner.jpg</url>
	<title>6box</title>
	<link>https://6box.me/</link>
	<description>Visit 6box - 6box</description>
</image>
```

But with nzb.su's information instead.

What I need is just the top section of the xml. Remove your api key, then email it to

`watchernzb@gmail.com`

I need to be able to find the name in their xml since it apparently doesn't follow the formatting of anything else I've seen.

**Comment by dg0664** [comment]
This is what I get:

You're using the wrong domain for your API / RSS calls

*Please use api.nzb.su <http://api.nzb.su>*



we are no longer redirecting nzb.su or www.nzb.su to api.nzb.su for you.

**Comment by nosmokingbandit** [comment]
Thanks for the invite @f3st3r113 !

With that I can troubleshoot much better since I can make all sorts of stupid requests and try to break it.

I'll get a permanent fix for you fairly soon.

**Comment by f3st3r113** [comment]
No Worries, thanks for the app!

**Comment by nosmokingbandit** [comment]
Well, apparently I can't. I don't have the extra cash to buy api access.

But.

Try this for me

`https://api.nzb.su/api?apikey=XXX&t=movie&imdbid=0947798`

Email the results in case there is anything sensitive in it.

**Comment by dg0664** [comment]
Sent to your email.

**Comment by claudeguy** [comment]
yes i also use nzb.su. and i think you already fixed the problem. i just downloaded the latest updates and now i can edit all movies! :-) thanks a lot!

**Comment by f3st3r113** [comment]
updated (on Ubuntu) now working as well.  Thank You!

**Comment by nosmokingbandit** [comment]
Commit https://github.com/nosmokingbandit/watcher/commit/393c673fbeb658fc917a59053b98ff5015e74265 has a better fix for this.

The initial fix took care of the symptoms but not the cause. After this commit, the `Source` field in the search results list will fill with the indexer's `link` if there is no `title` in the xml.

In simpler terms, instead of `Source: NZB.su` you'll end up seeing `Source: https://api.nzb.su/`
